app/_locales/en/messages.json

```diff
@@ -4814,6 +4814,10 @@
   "received": {
     "message": "Received"
   },
+  "receivingAddress": {
```

Comment:
Adding locales for page title when accessing address list page from Receive action button on home page